snow/consensus/snowman/network_test.go

```diff
@@ -74,26 +74,20 @@ func (n *Network) AddNode(t testing.TB, sm Consensus) error {
 	}
 
 	n.shuffleColors()
-	deps := map[ids.ID]ids.ID{}
 	for _, blk := range n.colors {
-		myDep, found := deps[blk.ParentV]
-		if !found {
-			myDep = blk.Parent()
-		}
 		myBlock := &snowmantest.Block{
 			Decidable: snowtest.Decidable{
 				IDV:    blk.ID(),
 				Status: blk.Status,
 			},
-			ParentV: myDep,
+			ParentV: blk.ParentV,
 			HeightV: blk.Height(),
 			VerifyV: blk.Verify(context.Background()),
 			BytesV:  blk.Bytes(),
 		}
 		if err := sm.Add(myBlock); err != nil {
```

Comment:
I think we can simplify this copy further by doing:
```golang
copiedBlk := *blk
if err := sm.Add(&copiedBlk); err != nil {
```

Reply:
Correct, done!